Resources/Prototypes/Entities/Objects/Weapons/Melee/baguette.yml

```diff
@@ -0,0 +1,17 @@
+- type: entity
+  name: baguette
+  parent: FoodBreadBaguette
+  id: WeaponBaguette
+  description: Bon appétit!
+  suffix: Weapon
+  components:
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    damage:
+      types:
+        Blunt: 16
+    soundHit:
+      path: /Audio/Weapons/bladeslice.ogg
```

Comment:
If it deals blunt, why the slicing sound?

Reply:
Good catch, I originally had it Blunt but meant to move it to slash. I must somehow dropped that from the diff. Derp.